barcodes_generator_abstract/readme/CONFIGURE.rst

```diff
@@ -4,20 +4,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: ../static/description/barcode_rule_tree.png
+.. image:: /barcodes_generator_abstract/static/description/barcode_rule_tree.png
```

Comment:
It was correct as previously. Why are you changing it?

Reply:
It must be pre-commit changing it.

Reply:
No, pre-commit doesn't touch this.